libs/ui/src/lib/notice/notice.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="cvi-notice__container cvi-notice--severity-{{ severity }}">
```

Comment:
The same DOM node can only have classes corresponding to the same BEM element, eg:

```html
cvi-notice__container cvi-notice__container--severity-{{ severity }}
```

or 

```html
<!-- Must be set on host element, eg via @HostBinding -->
cvi-notice cvi-notice--severity-{{ severity }}
```

In this case, I would advise to just move `cvi-notice--severity-{{ severity }}` to `@HostBinding` – it is convenient to set a BEM modifier class on a host element in an Angular component so that any of the nested ("element") classes could be later restyled according to different `severity` prop values (for example, with `#{$base}--severity-danger #{$base}__container {border-color: red; }`).